content/general/concepts/middleware/middleware.md

```diff
@@ -0,0 +1,66 @@
+---
+Title: 'Middleware'
+Description: 'Middleware is software that connects data, APIs, software tools, and other applications in order to bring a complete application to the end user.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'APIs'
+  - 'Database'
+  - 'Middleware'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/back-end-engineer-career-path'
+---
+
+Middleware is software that connects applications together. The term "middleware" comes from the fact that this software sits between the front-end, or client-side, of an application and the back-end, or server-side, where the resources are being requested.
+
+Data, APIs, software tools, and other applications are connected by middleware in order to bring a complete or bundled application to the end user. There are many types of middleware software depending on the use cases. Some of the most commonly used middleware is listed below.
```

Comment:
Thinking the last two sentences here can be shifted to after the "Types of Middleware" heading and before "Message Oriented Middleware"